client/src/api/netflixApi.test.js

```diff
@@ -1,4 +1,6 @@
 import { fetchMovies } from "./netflixApi";
+import config from "../../config.json"
+const {apiUrl} = config
```

Comment:
Api url is not secret information so it can be safely stored in process.env files